pyproject.toml

```diff
@@ -49,7 +49,7 @@ dependencies=[
     "resdata",
     "fastapi < 0.100.0",
     "filelock",
-    "iterative_ensemble_smoother>=0.1.1",
+    "iterative_ensemble_smoother@git+https://github.com/equinor/iterative_ensemble_smoother.git@main",
```

Comment:
Before merge I think we should push a new version of `iterative_ensemble_smoother`.

Reply:
Agreed.